manifests/charts/istiod-remote/values.yaml

```diff
@@ -45,3 +77,46 @@ istiodRemote:
 # Revision is set as 'version' label and part of the resource names when installing multiple control planes.
 revision: ""
 
+telemetry:
+  enabled: true
```

Comment:
should this be false by default?  and users can enable it when it is config cluster.

Reply:
fixed